src/Mvc/Mvc.Core/src/Resources.resx

```diff
@@ -503,5 +503,8 @@
   </data>
   <data name="Property_MustBeInstanceOfType" xml:space="preserve">
     <value>Property '{0}.{1}' must be an instance of type '{2}'.</value>
-  </data>  
-</root>
+  </data>
+  <data name="ObjectResultExecutor_MaxEnumerationExceeded" xml:space="preserve">
+    <value>{0} execeeded the maximum configured enumeration limit '{1}' when enumerating async enumerable type '{2}'.</value>
```

Comment:
Should we add more details here?

This doesn't address how a user would solve a problem if they have one.

I think it would be nice to say something like:

```
The '{formatter}' reached the configured maximum size of the buffer when an enumerating a value of type `FooType'. This limit is in place to prevent infinite streams of `IAsyncEnumerable<>` from continuing indefinitely. If this is not a programming mistake, consider ways to reduce the collection or size, or consider manually converting 'FooType' into a list rather than increasing the limit. 
```